src/scores/__init__.py

```diff
@@ -5,5 +5,6 @@
 import scores.continuous
 import scores.probability
 import scores.sample_data
+import scores.functions
 
 __version__ = "v0.0.2"
```

Comment:
With the new features, does it make sense to bump the version?

Reply:
Good call. I don't think I'll do it just exactly yet, but I might see if I can create a roadmap in github, and bump the version when things are merged to main.